destreamer.ts

```diff
@@ -156,8 +156,11 @@ async function rentVideoForLater(videoUrls: string[], outputDirectory: string, u
 
         // Add random index to prevent unwanted file overwrite!
         let k = 0;
-        while (fs.existsSync(outputDirectory+"/"+title+".mp4"))
-            title += ' - '+(++k).toString();
+        let ntitle = title;

```

Comment:
Ok i see, it kept appending " - *number*" to the title instead of changing the number, right? (asking just to be sure of the fix you applied)

Reply:
Yes, sorry